Makefile

```diff
@@ -103,8 +103,9 @@ quality-docker:
 coverage-docker:
 	docker run --rm -u root -v `(pwd)`:/edx/app/analytics_pipeline/analytics_pipeline -it edxops/analytics_pipeline:latest coverage xml
 
-coverage-local: test-local
+coverage-local:
 	python -m coverage html
```

Comment:
I think we can use single command to accomplish what all these `coverage` commands are doing except the `diff-cover`. Could you please try?
`coverage run -m pytest html xml -o coverage.xml`